hyperspy/tests/component/test_lorentzian.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+# Copyright 2007-2016 The HyperSpy developers
+#
+# This file is part of  HyperSpy.
+#
+#  HyperSpy is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+#  HyperSpy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with  HyperSpy.  If not, see <http://www.gnu.org/licenses/>.
+
+import numpy as np
+from numpy.testing import assert_allclose
+
+from hyperspy.components1d import Lorentzian
+
+
+def test_function():
+    g = Lorentzian()
+    g.A.value = 1.5*np.pi
+    g.gamma.value = 1
+    g.centre.value = 2
+    assert_allclose(g.function(2), 1.5)
+    assert_allclose(g.function(4), 0.3)
+
+def test_util_gamma_getset():
+    g1 = Lorentzian()
+    g1.gamma.value = 1.0
```

Comment:
1.0 is the default value, maybe it'll be good to set it to a different value?

Reply:
Good idea. Done. Added also a getset-test for fwhm.